packages/babel-helper-builder-react-jsx/src/index.js

```diff
@@ -172,6 +172,14 @@ You can turn on the 'throwIfNamespace' flag to bypass this warning.`,
     let _props = [];
     const objs = [];
 
+    const useSpread = file.opts.useSpread || false;
```

Comment:
I don't like that this accepts `0`, but let's keep it as-is for consistency with the other options of this file 🤷‍♀️

We should revisit it in Babel 8.

Reply:
IMO, we could begin on `useSpread` now because it is a new option. `useBuiltins: 0` is strange enough and it does not justify that you can even use `useSpread: 0`.